.storybook/mdx/highlight/highlight.css

```diff
@@ -1,7 +1,7 @@
 .highlight {
-    background-color: var(--o-ui-bg-alias-accent-light);
+    background-color: var(--hop-primary-surface-weak);
```

Comment:
I would use the background, border and text color of this:
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/ec3a4547-879a-4515-9875-ef1823e528be)
The banner component in hopper